main.js

```diff
@@ -0,0 +1,46 @@
+class Game {
+board = [[5, 4, 3, 2, 1],[],[]];
+
+//using map  to create a starting board mathod within the game class/object
+ startBoard() {this.board = this.board.map(function(towers){
```

Comment:
this.board should be on the next line review style guide: https://www.w3schools.com/js/js_conventions.asp